Harvest.Web/ClientApp/src/Tickets/TicketContainer.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useEffect, useState } from "react";
+import { useHistory, useParams } from "react-router-dom";
+import { Project, Ticket } from "../types";
+import { ProjectHeader } from "../Requests/ProjectHeader";
+import DatePicker from "react-date-picker";
+import { Button, FormGroup, Input, Label } from "reactstrap";
+
+interface RouteParams {
+  projectId?: string;
+}
+
+export const TicketContainer = () => {
+  const { projectId } = useParams<RouteParams>();
+  const [project, setProject] = useState<Project | undefined>();
```

Comment:
@jSylvestre curious as to why you did this?  If you do `useState<Project>()` since you leave the initializer empty it knows the `project` is possibly undefined and you don't need to add the `| undefined` part

Reply:
Was just coping code. Fixed.